functional/vfio/run.sh

```diff
@@ -107,19 +118,40 @@ setup_configuration_file() {
 		fi
 	done
 
+	if [ "$hypervisor" = "cloud-hypervisor" ]; then
+		CONFIG_FILE="${tmp_data_dir}/${hypervisor_clh}"
+		cp "${config_options_path}/${hypervisor_clh}" "${CONFIG_FILE}"
+	elif [ "$hypervisor" = "qemu" ]; then
+		CONFIG_FILE="${tmp_data_dir}/${hypervisor_qemu}"
+		cp "${config_options_path}/${hypervisor_qemu}" "${CONFIG_FILE}"
+	fi
+
+	# machine type applies to configuration.toml and configuration-qemu.toml
+	if [ "$hypervisor" != "cloud-hypervisor" ]; then
+		if [ $machine_type = "pc" ] || [ $machine_type = "q35" ]; then
```

Comment:
and microvm ? :smiley_cat:

Reply:
ignore this, I think microvm doesn't support vfio hotplug

Reply:
Might be worth adding a comment stating that.